app/apps/server/bridges/users.ts

```diff
@@ -4,7 +4,7 @@ import { IUserCreationOptions, IUser } from '@rocket.chat/apps-engine/definition
 
 import { setUserAvatar, checkUsernameAvailability, deleteUser, _setStatusTextPromise } from '../../../lib/server/functions';
 import { Users } from '../../../models/server';
-import { Users as UsersRaw } from '../../../models/server/raw';
+import { Subscriptions, Users as UsersRaw } from '../../../models/server/raw';
 import { AppServerOrchestrator } from '../orchestrator';
 
 export class AppUserBridge extends UserBridge {
```

Comment:
should it be Subscription bridge? I think we can do a lot of things related to the subscriptions like getting translation language